charts/rstudio-workbench/templates/configmap-session.yaml

```diff
@@ -5,5 +5,17 @@ metadata:
   name: {{ include "rstudio-workbench.fullname" . }}-session
   namespace: {{ $.Release.Namespace }}
 data:
-{{ include "rstudio-library.config.ini" .Values.config.session | indent 2 }}
+  {{- include "rstudio-library.config.ini" .Values.config.session | nindent 2 }}
+
+{{- /* if session/target namespace differs - duplicate session configuration there */}}
```

Comment:
What values do we need? What scenario are we covering? Could we add this to the comments here?

Reply:
Yes! I'll add color to the comment. The "session" config is mounted onto the session pods automatically per https://github.com/rstudio/helm/blob/27cb5989fde1c6b1049e515f0d2b87b8560d01a9/charts/rstudio-workbench/values.yaml#L10 and https://github.com/rstudio/helm/blob/27cb5989fde1c6b1049e515f0d2b87b8560d01a9/charts/rstudio-workbench/templates/configmap-general.yaml#L6-L12

This breaks if "targetNamespace" != "release namespace" because configmaps cannot be mounted across namespaces